packages/lexical-table/src/LexicalTableNode.ts

```diff
@@ -59,13 +82,24 @@ function setRowStriping(
 export class TableNode extends ElementNode {
   /** @internal */
   __rowStriping: boolean;
+  __colWidths?: number[];
 
   static getType(): string {
     return 'table';
   }
 
+  getColWidths() {
+    const self = this.getLatest();
+    return self.__colWidths;
+  }
+
+  setColWidths(colWidths: number[]) {
+    const self = this.getWritable();
+    self.__colWidths = colWidths;
+  }
```

Comment:
By convention setters are chainable in Lexical (this isn't totally consistent yet)

```suggestion
  setColWidths(colWidths: number[]): this {
    const self = this.getWritable();
    self.__colWidths = colWidths;
    return self;
  }
```